### ISSUE TITLE
Liquid Exchange Default Precision Incorrect Causing Values to be Truncated to Zero

### ISSUE THREAD

**Issue by lostless13** [ccxt/ccxt#5743]
MUST READ THIS BEFORE SUBMITTING ISSUES (read the link, then delete this message before submitting):

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

Please make sure your local version of ccxt is up to date with this repo. You can check by comparing the output of `ccxt.version` to https://github.com/ccxt/ccxt/blob/master/package.json#L3

- OS: Windows
- Programming Language version: javascript - node v10.15.3
- CCXT version: 1.18.1094
- Exchange: Liquid
- Method: createOrder (others probably affected as well)

Default precision for rate in this market is 5 resulting in the order getting set to zero by the priceToPrecision function. The price below shown is very close to the current market value. I've seen various other instances where the prices are not accurate.

```
var ccxt = require('ccxt')

let liquid = new ccxt.liquid({
    'apiKey': "deleted",
    'secret': "deleted"
})
let response = await liquid.createOrder("ENJ/BTC", "limit", "buy", 515.4639175257732, "0.00000388")
console.log(response)

```

Error output:
"InvalidOrder: liquid {"errors":{"price":["must_be_positive"]}}
    at liquid.handleErrors (c:\Users\sb\Documents\code\CryptoTrader\node_modules\ccxt\js\liquid.js:820:31)
    at response.text.then (c:\Users\sb\Documents\code\CryptoTrader\node_modules\ccxt\js\base\Exchange.js:629:18)
    at process._tickCallback (internal/process/next_tick.js:68:7)"

**Comment by kroitor** [comment]
In cases when you supply values that don't satisfy exchange's requirements on min limits and precision, the CCXT library will return InvalidOrder to you. Which is exactly the case. You should respect the exchange limitations, as described in the Manual here: 

- https://github.com/ccxt/ccxt/wiki/Manual#precision-and-limits

Let us know if that does not answer the issue.

**Comment by kroitor** [comment]
This should be fixed as of CCXT 1.18.1095. The new build will arrive in 15 minutes. Let us know if that does not help. Thx for reporting!
